ParabolInc/parabol
# chore: Show link to guide on Jira auth popup close

# Description

Fixes #7756 
When the user closes the popup without finishing authorization for Jira, show an error message with a link to our troubleshooting guide.

## Demo

https://www.loom.com/share/6bfa60b7e9ee42f2b9f8231a1d59d85b?sid=da869b36-7d01-4856-9006-8bc93b7bc211

## Testing scenarios

- try integrating from integration settings, Sprint Poker, task dashboard
- close the popup
- see the error

## Final checklist
